lib/pgbackups-archive/storage.rb

```diff
@@ -19,7 +19,12 @@ def connection
   end
 
   def bucket
-    connection.directories.get ENV["PGBACKUPS_BUCKET"]
+    directory = connection.directories.get ENV["PGBACKUPS_BUCKET"]
```

Comment:
I like the namespaced concept here.  The method should probably be renamed to `directory` or `path` though, since it no longer returns simply a bucket name.